cookbooks/nginx/attributes/default.rb

```diff
@@ -7,3 +7,5 @@
                                      # Mask nginx on non-app instances
                                      true
                                    end
+default['nginx']['syslog'] = false
+default['nginx']['syslog_path'] = 'syslog:server=unix:/var/log/nginx.sock'
```

Comment:
@wildbillcat That's not the correct path, the syslog daemon socket is at `/dev/log` (http://manpages.ubuntu.com/manpages/bionic/man8/syslogd.8.html).